Prace_domowe/Praca_domowa3/Jamrozy_Bartosz/pd3_australian_weather.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>Bardzo dobrze, że to sprawdzono!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>